ShijiLighting/GHIElectronics.TinyCLR.Drivers.ShijiLighting.APA102C/APA102C.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Drawing;
+using GHIElectronics.TinyCLR.Devices.Spi;
+
+namespace GHIElectronics.TinyCLR.Drivers.ShijiLighting.APA102C
+{
```

Comment:
Brace style

Reply:
I missed that one when I fixed the others. :( (see commit 9225bd9)